### DISCUSSION TITLE
Any fork supporting Python 2.7?

### DISCUSSION THREAD

**Discussion by ReenigneArcher** [pradyunsg/furo#583] [Questions] [answered]
I've been in the process of switching over all of my project docs from RTD theme to furo, unfortunately I have two projects which must remain in Python 2.7 due to them being plugins for Plex which is stuck on 2.7.

Is anyone aware of a fork of furo that may support Python 2.7? This would be for `Sphinx==1.8.6`.

**Comment by pradyunsg** [comment, marked as answer]
Not that I'm aware of, no. You might want to explore if the documentation can be built with Python 3, given that Python 2 has been EoL for almost two full years now, after an extended life period by five years to facilitate migrating off of it.
